sdk/search/azure-search-documents/azure/search/documents/_api_versions.py

```diff
@@ -17,8 +20,8 @@ def validate_api_version(api_version):
         raise ValueError("Unsupported API version '{}'. Please select from:\n{}".format(api_version, versions))
 
 
-def get_api_version(kwargs, default):
+def get_api_version(kwargs):
```

Comment:
hey @xiangyan99 idt you need most of this file once you've generated multiapi code. One thing that TA added is an API version [enum](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_base_client.py#L13), so you might want to add one as well

Reply:
@xiangyan99 should've been more clear in previous message. I really don't think you need any of `_validate_api_version`, `get_api_version`, that should all be handled by the multiapi client